src/filebrowser/sharedlink-dialog.cpp

```diff
@@ -16,18 +17,25 @@ SharedLinkDialog::SharedLinkDialog(const QString &text, QWidget *parent)
     QVBoxLayout *layout = new QVBoxLayout;
 
     QLabel *label = new QLabel(tr("Share link:"));
+    label->setFont(changeFontSize(label->font(), 12));
     layout->addWidget(label);
     layout->setSpacing(5);
     layout->setContentsMargins(9, 9, 9, 9);
 
-    QLineEdit *editor = new QLineEdit;
-    editor->setText(text_);
-    editor->selectAll();
-    editor->setReadOnly(true);
-    layout->addWidget(editor);
+    editor_ = new QLineEdit;
+    editor_->setText(text_);
+    editor_->selectAll();
+    editor_->setReadOnly(true);
+    layout->addWidget(editor_);
 
     QHBoxLayout *hlayout = new QHBoxLayout;
 
+    QCheckBox *is_download_checked = new QCheckBox(tr("Direct Download"));
+    connect(is_download_checked, SIGNAL(stateChanged(int)),
+            this, SLOT(onDownloadStateChanged(int)));
+    is_download_checked->setFont(changeFontSize(is_download_checked->font(), 10));
```

Comment:
It costs at least two lines of code to set font size in style sheet (`QObject::setObjectName` and `font-size:xxx`), while it costs only one loc (line of code) to set it in `C++`.